### DISCUSSION TITLE
createIssuedDocument visualizzare fattura creata

### DISCUSSION THREAD

**Discussion by GianmariaTS** [fattureincloud/api#221] [Q&A] [answered]
Buonasera,

ho utilizzato la funzione createIssuedDocument e la risposta all'invio non da errore ma non riesco a visualizzare la fattura nella console, sicuramente è generata in quanto non la riesco ad inviare una seconda volta.

La connessione che faccio avviene con un token generato nelle impostazioni - applicazioni collegate.

Forse potreste dirmi se ho dimenticato qualche passaggio oppure come poter visualizzare la fattura inviata?

Grazie,
G.

**Comment by MattiaMarchiorato** [comment, marked as answer]
@GianmariaTS se la fattura è stata creata la trovi nella console di fatture in cloud da interfaccia web.

**Reply by valmoz** [reply]
Ciao @GianmariaTS 
prima di tutto, se hai ricevuto una risposta 200 OK la creazione dovrebbe essere andata a buon fine. Ti chiederei di verificare la risposta che ottieni quando chiami il metodo in modo da identificare se la richiesta è andata in successo o in errore.

Se la risposta è andata in successo, come detto da @Mattia-Marchiorato (a proposito, grazie) la fattura dovrebbe essere visibile da interfaccia Web.

In alternativa, da API puoi:

- utilizzare l'ID inviato tramite risposta per effettuare una GetIssuedDocument
- utilizzare la ListIssuedDocuments per cercare la fattura

Grazie
Mauro

**Reply by GianmariaTS** [reply]
@Mattia-Marchiorato e @valmoz vi ringrazio ora le ho trovate.
G.
